doc/authoring_help.md

```diff
@@ -217,15 +217,15 @@ The first example is only supported on the profile `latest` and above whilst the
     examples:
         - summary: Create a storage account MyStorageAccount in resource group MyResourceGroup in the West US region with locally redundant storage.
           command: az storage account create -n MyStorageAccount -g MyResourceGroup -l westus --sku Standard_LRS
-          min_profile: latest
+          supported_profiles: latest, 2018-03-01-hybrid
```

Comment:
This raises an interesting point. We *don't* want to have to find and update this example every time a new profile is minted or it will "disappear". Perhaps it would make sense to have an "unsupported_profiles" kwarg for this like this one where the value is "2017-03-09-profile", basically saying that the example works on all profiles except that one listed. In that way, it does not add to the maintenance burden.

Reply:
So in actuality, this is an addition (and not a conversion). I would have left it as `latest` alone. However, I had realized the example was valid for both `latest` and `hybrid`, and so I added the hybrid profile.  I tested the example out on the hybrid profile and was able to create an LRS storage account, and so thought I'd modify the example.

Reply:
Right, so you just proved my point. Thank you :)

This example would "disappear" on the hybrid profile even though it is really applicable to everything after 2017-03-09. You recognized that the example was applicable and added it. Every new profile that gets minted and has a value here will either need to go through and add itself to each of these or be "lost". Might be a good talking point at scrum if we can ever get back into the office.

Reply:
Oh I see! Sorry, I misunderstood you at first.  You were talking about future profiles not this one in particular. Yeah it would be nice to talk with the rest of the team about this.

Reply:
This may be something we can build tooling to help with. If the examples are setup to describe the API Versions of different services we rely on, we can reverse the problem and use a profile definition to find all relevant examples. Even if we still have to serialize it into this format, it would still ensure that we have a way of auditing ourselves.